src/main/java/delete_event_use_case/DeleteEventDataAccess.java

```diff
@@ -0,0 +1,33 @@
+package delete_event_use_case;
+
+import data_access.DataAccessDay;
+import data_access.DataAccessEvent;
+import data_access.WeekDataAccess;
+
+import java.util.HashMap;
+import java.util.ArrayList;
+
+/**
+ * Delete Event Data Access class.
+ * @author : Khizer Ahmad
+ * @created: November 19th 2022
+ * @since : 2022-11-21, Mon.
+ **/
```

Comment:
Please move the documentation above the package to maintain consistency throughout everyone's files.